ambry-api/src/main/java/com.github.ambry/account/Container.java

```diff
@@ -335,6 +344,17 @@ private Container(JSONObject metadata, short parentAccountId) throws JSONExcepti
         mediaScanDisabled = metadata.optBoolean(MEDIA_SCAN_DISABLED_KEY, MEDIA_SCAN_DISABLED_DEFAULT_VALUE);
         replicationPolicy = metadata.optString(REPLICATION_POLICY_KEY, null);
         ttlRequired = metadata.optBoolean(TTL_REQUIRED_KEY, TTL_REQUIRED_DEFAULT_VALUE);
+        JSONArray contentTypeWhitelistForFilenamesOnDownloadJson =
+            metadata.optJSONArray(CONTENT_TYPE_WHITELIST_FOR_FILENAMES_ON_DOWNLOAD);
+        if (contentTypeWhitelistForFilenamesOnDownloadJson != null) {
+          contentTypeWhitelistForFilenamesOnDownload = new HashSet<>();
+          for (int i = 0; i < contentTypeWhitelistForFilenamesOnDownloadJson.length(); i++) {
```

Comment:
`JSONArray` is now `Iterable`, so you can do 
```java
for (Object contentType : contentTypeWhitelistForFilenamesOnDownloadJson) {
  set.add(contentType.toString());
}
```

Reply:
Done